### PR TITLE
Prepare release v0.1.3

### PR DESCRIPTION
**Pull request by krebernisak**

### CODE REVIEW

**Review by boxhock — APPROVED** [review]
I like the new error handling!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,16 +6,30 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+  - `reduce` to reduce an array to a result. Different reducers are supported like: `sum`, `product`, `average`, `median`, `min` & `max`.
+  - `trueusd` to query supply and issuance data for TrueUSD.
+  - `cryptoid` to get chain difficulty measurement from different networks.
+  - `blockchair` to get chain difficulty measurement from different networks.
+  - `bitso` to get last 24 hours volume weighted average price (vwap) for multiple markets.
+  - `cryptomkt` to get last transaction price for multiple markets.
+  - `satoshitango` to get last bid price for multiple markets.
+  - `bitex` to get volume weighted average price (vwap) for multiple markets.
+  - `orchid-bandwidth` to get available bandwidth from Chainlink as service provider.
+  - `genesis-volatility`
```
Comment (justinkaseman) [inline CHANGELOG.md:33]:
```suggestion
  - `covid-tracker` -  to get query COVID-19 statistics.
  - `lition` - to query the price in Euros per MWh for Lition.
  - `coinlore`- to query the market dominance and market capitlization for Bitcoin.
  - `messari` - to query the marker dominance for Bitcoin.
```
We should document the new adapters I made here too. I'll make sure to add them myself next time.

Comment (krebernisak) [reply CHANGELOG.md:33]: Thanks! 

Don't worry, keeping track of the changes is something we introduced just recently so we all need to get used to it. But I think it should be useful.

**Review by justinkaseman — APPROVED** [review]
Might want to add the description to the Genesis adapter, otherwise LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +30,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
   - `bitex` to get volume weighted average price (vwap) for multiple markets.
   - `orchid-bandwidth` to get available bandwidth from Chainlink as service provider.
   - `genesis-volatility`
```
Comment (justinkaseman) [inline CHANGELOG.md:32]:
```suggestion
  - `genesis-volatility` to get implied volatility for multiple markets.
```

Comment (krebernisak) [reply CHANGELOG.md]: Thanks!

**Review by boxhock — APPROVED** [review]

**Pull request merged**